src/ngraph/frontend/onnx_import/ops_bridge.cpp

```diff
@@ -17,13 +17,13 @@
 #include <algorithm>
 #include <functional>
 
-#include "attribute.hpp"
 #include "ngraph/frontend/onnx_import/op/add.hpp"
```

Comment:
get rid of `ngraph/frontend/onnx_import/` from `#include`. Leave only `op/add.hpp`.